modules/fedimint-ln-server/src/lib.rs

```diff
@@ -748,23 +748,34 @@ impl ServerModule for Lightning {
         vec![
             api_endpoint! {
                 "/account",
-                async |module: &Lightning, context, contract_id: ContractId| -> ContractAccount {
-                    module
+                async |module: &Lightning, context, contract_id: ContractId| -> Option<ContractAccount> {
+                    Ok(module
                         .get_contract_account(&mut context.dbtx(), contract_id)
-                        .await
-                        .ok_or_else(|| ApiError::not_found(String::from("Contract not found")))
+                        .await)
+                }
+            },
+            api_endpoint! {
+                "/wait_account",
+                async |module: &Lightning, context, contract_id: ContractId| -> ContractAccount {
+                    Ok(module
+                        .wait_contract_account(context, contract_id)
+                        .await)
                 }
             },
             api_endpoint! {
                 "/offer",
```

Comment:
are we always going to expose a wait and non-wait version of the endpoints?

Reply:
We can make it a parameter. 
We can also just remove the non wait version and make caller wrap it with a short timeout if it doesn't want to wait the full 60 seconds.

Reply:
Yeah I don't see why we don't always wait, let the client decide whether to timeout.  Would like to avoid the complexity of having 2 endpoints for every call.

Reply:
Lets do this in a follow and not block this.